### PR TITLE
[Ember Times Issue 66] Add section on Issue Triage

### PR DESCRIPTION
**Pull request by pixelhandler**
Add section to `[WIP] Ember Times Issue 65`, #3563

## What it does

Adds a section to issue 65 of Ember Times for Issue Triage effort for the past 2 weeks.

## Related Issue(s)

Merges into branch for #3563

### CODE REVIEW

**Comment by amyrlam** [comment]
Put on Issue #66 https://github.com/emberjs/website/pull/3577 instead

File: source/blog/2018-09-21-the-ember-times-issue-65.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  <a href="https://github.com/yohanmishkin" target="gh-user">@pixelhandler</a>.
+</p>
+<p>
+  The `@emberjs/issues-team` objective is to increase signal-to-noise ratio of issues
+  and 🐛 &nbsp;bugs so that the core team (and contributors) can take action to resolve bugs,
+  and spend less effort sorting the issues queue. Currently there are about 67 open bugs, of
+  those 25 had a reproduction (e.g. using <a href="https://ember-twiddle.com">ember-twiddle.com</a>)
+  , and 1 with a pull request - total count about 270. Our aim is to get down to 150 in the next
+  couple months. If you'd like to help see the
+  <a href="https://discordapp.com/channels/480462759797063690/480523776845414412">#issue-triage</a>
```
Comment (amyrlam) [inline source/blog/2018-09-21-the-ember-times-issue-65.md]: I didn't know you could link to a Discord channel. Do you mind sharing how you did that?

File: source/blog/2018-09-21-the-ember-times-issue-65.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  </a> over the past two weeks 🤩… 
+  <a href="https://github.com/yohanmishkin" target="gh-user">@yininge</a>,
+  <a href="https://github.com/ryanlabouve" target="gh-user">@ryanlabouve</a>,
+  <a href="https://github.com/yohanmishkin" target="gh-user">@yohanmishkin</a>,
+  <a href="https://github.com/esbanarango" target="gh-user">@esbanarango</a>,
+  <a href="https://github.com/hakilebara" target="gh-user">@hakilebara</a>, and
+  <a href="https://github.com/yohanmishkin" target="gh-user">@pixelhandler</a>.
+</p>
+<p>
+  The `@emberjs/issues-team` objective is to increase signal-to-noise ratio of issues
```
Comment (amyrlam) [inline source/blog/2018-09-21-the-ember-times-issue-65.md]: Looks like single tick doesn't have formatting, consider italics instead?

**Comment by amyrlam** [comment]
Thanks for taking the initiative on issue triage, and for writing about it! 😄

I tried to change the base branch but has a lot of other stuff, may be easier to just make a new branch to https://github.com/emberjs/website/pull/3577 instead if not too much trouble!

File: source/blog/2018-09-21-the-ember-times-issue-65.md (hunk 1 above)
Comment (pixelhandler) [reply source/blog/2018-09-21-the-ember-times-issue-65.md]: @amyrlam I logged into discord on the web and just grabbed the URL.

File: source/blog/2018-09-21-the-ember-times-issue-65.md (hunk 2 above)
Comment (pixelhandler) [reply source/blog/2018-09-21-the-ember-times-issue-65.md]: yeah italics works :)

**Comment by pixelhandler** [comment]
Gonna close this and reopen with new branch

**Pull request closed without merging**